setup.py

```diff
@@ -7,37 +7,34 @@
 
 here = os.path.abspath(os.path.dirname(__file__))
 
-install_requires = [
-    'pykube',
-    'click==7.0',
-    'dask[complete]',
-    'toolz',
-    'fastparquet',
-    'flask==1.0.2',
+min_requires = [
     'google-auth-httplib2',
     'google-auth',
     'google-cloud-core>=1.3.0',
     'google-cloud-storage',
-    'joblib==0.11',
-    'luigi',
-    'numpy==1.16.3',
-    'pandas==0.23.4',
+    'joblib>=0.11',
+    'numpy>=1.16.3',
+    'pandas>=0.23.4',
```

Comment:
this could not be optional also? like ai-toolkit? this is so heavy on install.

Reply:
hum... I think most of the people that use pycarol today they use to fetch parquet, that is pandas dependent. Maybe we could add a "lite" that is only the APIs..
but still we dont have that much use for APIs that are not involving data.

Reply:
done